recipes/default.rb

```diff
@@ -24,6 +24,8 @@
   ports %w(9323)
 end
 
+package %w(cron)
```

Comment:
This unfortunately doesn't work with CentOS 7 as the package name doesn't. We already pull this in via ``base::default`` but I'd rather we didn't pull that completely in. Instead let's do the following:
1. Add a dep for the `cron` cookbook
2. Add the following resources:
```ruby
cron_package 'osl-docker'
cron_service 'osl-docker' do
  action [:enable, :start]
end
```

This should in theory work on all supported platforms and won't conflict with what we do in base.

Please make sure you update the tests as well.